package.json

```diff
@@ -182,6 +182,7 @@
         "start-tls": "npm run webapp:dev-ssl",
         "serve": "npm run start --",
         "build": "npm run webapp:prod --",
+        "jest": "jest --coverage --log-heap-usage --max-workers=4 --config jest.config.js",
```

Comment:
Why do we need the extra definition here? Why isn't it enough to have one test? It also handles the coverage and specifies the workers as well as printing the heap usage.